remappers/IHumanStateRemapper/IHumanStateRemapper.h

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (C) 2020 Istituto Italiano di Tecnologia (IIT)
+ * All rights reserved.
+ *
+ * This software may be modified and distributed under the terms of the
+ * GNU Lesser General Public License v2.1 or any later version.
+ */
+
+#ifndef HDE_DEVICES_IHUMANSTATEREMAPPER
+#define HDE_DEVICES_IHUMANSTATEREMAPPER
+
+#include "IHumanState.h"
+
+#include <yarp/dev/DeviceDriver.h>
+#include <yarp/dev/PreciselyTimed.h>
+#include <yarp/os/PeriodicThread.h>
+#include <yarp/os/TypedReaderCallback.h>
+
+namespace human {
+    class HumanState;
+}
+namespace hde {
+    namespace devices {
+        class IHumanStateRemapper;
+    } // namespace devices
+} // namespace hde
+
+class hde::devices::IHumanStateRemapper final
+    : public yarp::dev::DeviceDriver
+    , public hde::interfaces::IHumanState
+    , public yarp::os::TypedReaderCallback<human::HumanState>
+    , public yarp::os::PeriodicThread
+    //, public yarp::dev::IPreciselyTimed
```

Comment:
Clean up

Reply:
Done in https://github.com/robotology/human-dynamics-estimation/pull/204/commits/2b4fafe9fe3e722b8719b7613bc4cc6d4cbf181b